django-backend/.gitignore

```diff
@@ -0,0 +1,129 @@
+data-formatter/*.json
+data-formatter/*.bak
+data-formatter/*.dat
+data-formatter/*.dir

```

Comment:
just add the extension, like `*.bak` .

Reply:
I have updated the gitignore and removed the files which are added by shelve i.e .bak, .dat and .dir. Didn't add .json to global gitignore as it will lead to an empty directory `(path: src/dataviz_api/data/)` and git will ignore it.